docs/product-user-guide/src/main/asciidoc/data-objects-create-proc.adoc

```diff
@@ -1,30 +1,30 @@
-[id='data_objects_create_proc']
-= Creating a Data Object
+[id='data-objects-create-proc']
+= Creating data objects
 
-The data objects that you define can be implemented in various types of assets in your project, and are essential building blocks for your project.
+The data objects that you define can be implemented in various types of project assets, such as in guided decision tables and guided rules, and are the building blocks for the assets in your project.
```

Comment:
Data objects **cannot** be _implemented_ in various types of project asset; but they are used/consumed by the types. Data Objects can only be _implemented_ with the Data Object Editor.